.github/workflows/certora_recovery.yml

```
@@ -0,0 +1,44 @@
+name: certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rule: ['verifyRecoveryModule.sh']
+    steps:
+      - uses: actions/checkout@v3
```

CHeckout is on v4 now:

```suggestion
      - uses: actions/checkout@v4
```

---

Updated 👍🏾